### ISSUE TITLE
🐛  Rowspan is not rendering correctrly

### ISSUE THREAD

**Issue by RafalWilinski** [inovua/reactdatagrid#175]
- what edition are you using - community 
- version for `@inovua/reactdatagrid-community`or `@inovua/reactdatagrid-enterprise` - `4.1.24`

Relevant code or config - demo from https://reactdatagrid.io/docs/cell-rowspan-and-colspan

```javascript
import ReactDataGrid from "@inovua/reactdatagrid-community";
import "@inovua/reactdatagrid-community/index.css";

import people from "./people";

const columns = [
  {
    name: "id",
    defaultWidth: 60,
    header: "Id",
    type: "number",
    colspan: ({ data, column, columns }: any) => {
      // make every other row cell expand for 2 columns if the next column is the age column
      if (
        data.id % 2 &&
        columns[column.computedVisibleIndex] &&
        columns[column.computedVisibleIndex + 1].name === "age"
      ) {
        return 2;
      }

      return 1;
    },
  },
  { name: "age", header: "Age", defaultFlex: 1, type: "number" },
  { name: "name", defaultFlex: 1 },
  {
    name: "country",
    header: "Country",
    defaultFlex: 1,
    rowspan: ({ value, dataSourceArray, rowIndex, column }: any) => {
      let rowspan = 1;

      const prevData = dataSourceArray[rowIndex - 1];
      if (prevData && prevData[column.name] === value) {
        return rowspan;
      }
      let currentRowIndex = rowIndex + 1;
      while (
        dataSourceArray[currentRowIndex] &&
        dataSourceArray[currentRowIndex][column.name] === value
      ) {
        rowspan++;
        currentRowIndex++;
        if (rowspan > 9) {
          break;
        }
      }
      return rowspan;
    },
  },
];

const App = () => {
  return (
    <div>
      <h3>Rowspan and colspan example</h3>
      <p>
        Try and sort the <b>COUNTRY</b> column to see the cells with same
        country spanning together.
      </p>
      <ReactDataGrid
        showZebraRows={false}
        style={gridStyle}
        columns={columns}
        dataSource={people}
      />
    </div>
  );
};

const gridStyle = { minHeight: 550 };
export default App;
```

What you did: Replicate the demo from https://reactdatagrid.io/docs/cell-rowspan-and-colspan

What happened - rowspans are not rendered properly:

![Screenshot 2021-08-16 at 15 40 46](https://user-images.githubusercontent.com/3391616/129574307-c22b63f0-92fe-4e5d-9908-6aa2dd7414de.png)

Looks like the row that should be taking "more rows" is being computed just fine but the rows below are rendering over it.

![CleanShot 2021-08-16 at 15 53 43](https://user-images.githubusercontent.com/3391616/129574859-cdd74163-106e-49ce-9bb8-0d687b340f7c.png)

Reproduction repository: https://github.com/RafalWilinski/reactdatagrid-rowspan-problem/tree/master/src

**Comment by RafalWilinski** [comment]
Running an example directly from http://github.com/inovua/reactdatagrid/blob/master/examples/pages/rowspan-and-colspan/index.page.tsx

Also yields the following rendering error:

![Screenshot 2021-08-16 at 18 57 59](https://user-images.githubusercontent.com/3391616/129601192-aa7b2d28-5e7c-402a-b71a-ff0636b1d8b6.png)

**Comment by inovua-admin** [comment]
Fixed in version `5.3.0`.
